README.md

```diff
@@ -83,7 +93,9 @@ You will need `kubectl` on your PATH and configured to control minikube in order
 
 ### Installing the Konveyor operator
 
-The [konveyor/operator git repository](https://github.com/konveyor/operator) provides a script to install Tackle locally using `kubectl`. You can [inspect its source here](https://github.com/konveyor/operator/blob/main/hack/install-tackle.sh). This script creates the `konveyor-tackle` namespace, CatalogSource, OperatorGroup, Subscription and Tackle CR, then waits for deployments to be ready.
+Follow the official instructions for [Installing Konveyor Operator](https://konveyor.github.io/konveyor/installation/#installing-konveyor-operator)
+
+Alternatively, the [konveyor/operator git repository](https://github.com/konveyor/operator) provides a script to install Tackle locally using `kubectl`. You can [inspect its source here](https://github.com/konveyor/operator/blob/main/hack/install-tackle.sh). This script creates the `konveyor-tackle` namespace, CatalogSource, OperatorGroup, Subscription and Tackle CR, then waits for deployments to be ready.
```

Comment:
I think for any new text, we should refer to "Tackle" as "Konveyor". I realize that the waters do get muddy since the repo is still titled "tackle-2". 


"provides a script to install ~Tackle~ Konveyor locally using `kubectl`. "

Reply:
Fine to disregard Tackle/Konveyor issue for now. Can push that out to be a future us problem.

Reply:
OK. I'm resolving the comment then.